patches/domino+2.1.6.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/node_modules/domino/lib/sloppy.js b/node_modules/domino/lib/sloppy.js
```

Comment:
I think the problem we were solving with the tests has already been resolved by the `update-dependencies` branch.

I went ahead and removed it locally, re-ran `yarn install` and `yarn test` and the tests were able to successfully run. Could you try the same and remove the patch file if it's all good for you as well @Paul-Clue?

Reply:
It worked! I removed the patch and pushed the changes.